api/graphql/mutations/responsibilities.js

```diff
@@ -0,0 +1,97 @@
+const { GraphQLYogaError } = require('@graphql-yoga/node')
+
+export async function addGroupResponsibility ({ groupId, title, description, userId }) {
+  if (!userId) throw new GraphQLYogaError('No userId passed into function')
+
+  if (groupId && title) {
+    const groupMembership = await GroupMembership.forIds(userId, groupId).fetch()
+    const responsibilities = await Responsibility.fetchForUserAndGroupAsStrings(userId, groupId)
+    if (groupMembership && (groupMembership.get('role') === GroupMembership.Role.MODERATOR || responsibilities.includes(Responsibility.constants.RESP_ADMINISTRATION))) {
```

Comment:
I think we should we can get rid of the role == MODERATOR part and update the database so all current moderators have ADMIN powers